Fix exit validator documentation links

hewison-chris opened the pull request.

hewison-chris committed:
Update the doc link for validator exiting

TrustHenry commented on validator/package/validator.yaml:
```diff
@@ -10,12 +10,12 @@ wallet-dir: /var/lib/prysm/validator
 
 # Optional tuning parameters
-# For full list, see https://docs.prylabs.network/docs/prysm-usage/parameters
+# For full list, see https://agora-cl-docs.bosagora.org/docs/prysm-usage/parameters
```
Sorry, Does not work the link.

hewison-chris replied:
Good catch, I forgot that some folders and pages were renamed.

hewison-chris replied:
I have dropped the commit with the other renames from this PR as it will be tackled by issue https://github.com/bosagora/AgoraMainNet/issues/80 where some pages will be removed and updated.

hewison-chris committed:
Change the log for insecure gRPC

The recommended configuration is to run both node and validator on the
same machine so insecure gRPC is expected so no need to make a Warn log.

MichaelKim20 reviewed (dismissed):
LGTM
After merging, I will make a docker image and test it.

hewison-chris commented:
Local test using local-testnet and Agora-cl command:
```
validator accounts voluntary-exit --chain-config-file chain-config.yaml --accept-terms-of-use -wallet-dir=wallet --beacon-rpc-provider 127.0.0.1:9001
```
In the chain-config-file:
`AGORASCAN_URL: http://localhost:3333/`

<img width="1056" alt="Screen Shot 2022-10-26 at 16 30 04" src="https://user-images.githubusercontent.com/11815575/197962518-f5e9e00c-306e-448b-b567-f849dd7add2c.png">

hewison-chris committed:
Enable correct messages when running Validator commands

MichaelKim20 reviewed (approved):
LGTM
